### PR TITLE
Assign a unique cluster name to each cluster based on the parameter --cluster-name-seed

### LINKED ISSUE #5: Unique identifier on cluster names
When creating clusters in a batch, all of them have the same name:

$ osde2e/out/osde2ectl list --custom-config custom_config.yaml
2020/11/19 12:14:44 Custom YAML config provided, loading from custom_config.yaml
NAME                     ID                                 STATE          STATUS                   OWNER                    INSTALLED VERSION        UPGRADE VERSION
morenod                  1h20e43ahroenhk414nd8gecet5ce6v1   pending        waiting-for-ready        dsanzmor                 openshift-v4.5.13
morenod                  1h20e4469vqf9s25e2kbohpv8qdru2kl   pending        waiting-for-ready        dsanzmor                 openshift-v4.5.13
morenod                  1h20e449genfgsg85as2c1p3ktct88aa   pending        waiting-for-ready        dsanzmor                 openshift-v4.5.13
morenod                  1h20e46v5lmkke8cl6lvo7jaau1skk6a   pending        waiting-for-ready        dsanzmor                 openshift-v4.5.13
morenod                  1h20e4et9rkrf9aedjd7m2ao7ghigtou   pending        waiting-for-ready        dsanzmor                 openshift-v4.5.13

### LINKED ISSUE #54: Initialize my_doc['log-metrics'] before create new fields
As we do with the entire document:

my_doc = {}

We need to initialize my_doc['log_metrics'] to {} before assigning new fields

### LINKED ISSUE #55: Include cluster identifiers in scale test console output
It would be super helpful if log lines like this:

```
10:07:36 2021-02-04 16:08:09.334 INFO osde2e-wrapper - _build_cluster: Started cluster 455
```

...could provide some kind of cluster identifier, e.g.

```
10:07:36 2021-02-04 16:08:09.334 INFO osde2e-wrapper - _build_cluster: Started cluster 455: osde2e-1wxyz
```

I'm given to understand this may be nontrivial because the process that spits out that message spawns the cluster creation asynchronously. But hey, wanted to get the idea down.

### PR DESCRIPTION
**Pull request by morenod**
- Added --cluster-name-seed parameter

Cluster name seed is limited to 6 chars because max long of cluster name is 15

seed (6) + "-" + rand(3) + "-" + cluster_index(4)

According to cluster_index, test is limited to 9999 cluster installation.

- Folders created for each cluster are now the cluster_name, instead of 1,2,3,...

- added log_metrics initialization on ES uploading function

### CODE REVIEW

File: osde2e-wrapper.py [hunk 1]
```diff
@@ -346,6 +350,10 @@ def main():
         '--cleanup',
         help='Should we delete the temporary directory',
         default=False)
+    parser.add_argument(
+        '--cluster-name-seed',
+        type=str,
+        help='Seed used to generate cluster names.')
```
Comment (chentex) [inline osde2e-wrapper.py]: Should we add here the note on the 6 character limit.

**Review by chentex — APPROVED** [review]
LGTM, just with a nit-pic.

**Comment by morenod** [comment]
@dry923 default value on cluster-name-seed added. Ready to merge (please squash commits, I added using the webui)

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**